src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1137,9 +1158,10 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger? errorLogger, Cancella
             try
             {
                 // NOTE: Unlike the PDB path, the XML doc path is not embedded in the assembly, so we don't need to pass it to emit.
-                var emitOptions = Arguments.EmitOptions.
-                    WithOutputNameOverride(outputName).
-                    WithPdbFilePath(PathUtilities.NormalizePathPrefix(finalPdbFilePath, Arguments.PathMap));
+                var emitOptions =
+                    Arguments.EmitOptions.
+                        WithOutputNameOverride(outputName).
+                        WithPdbFilePath(PathUtilities.NormalizePathPrefix(finalPdbFilePath, Arguments.PathMap));
```

Comment:
Can we undo this? Looks like style change only.

Reply:
Fixed: https://github.com/dotnet/roslyn/pull/62641/files#diff-96f63d80d819d3bcedfa76e09251cb606e8d7b6da593d2457ed85566a882276d

Reply:
Fixed: https://github.com/dotnet/roslyn/pull/62641/files#diff-96f63d80d819d3bcedfa76e09251cb606e8d7b6da593d2457ed85566a882276d